src/transactions/CreateAccountOpFrame.cpp

```diff
@@ -105,4 +105,12 @@ CreateAccountOpFrame::doCheckValid(Application& app, uint32_t ledgerVersion)
 
     return true;
 }
+
+std::unordered_set<LedgerKey>
+CreateAccountOpFrame::getLedgerKeysToPrefetch(Application& app)
+{
+    std::unordered_set<LedgerKey> keys{
+        getLedgerKey(mCreateAccount.destination)};
+    return keys;
```

Comment:
Nitpick: For brevity could just `return {getLedgerKey(mCreateAccount.destination)};`